scheduler/test/cook/test/compute_cluster.clj

```diff
@@ -192,6 +193,7 @@
            {:a {}}))))
 
 (deftest test-get-job-instance-ids-for-cluster-name
+  (setup)
```

Comment:
how does this relate to the rest of the PR?

Reply:
Unit tests need to run this to setup mount state, however, a lot omit it and piggyback on prior tests having had already run mount. I encountered this test breaking and fixed it.